configs/tdh-test.yml

```diff
@@ -0,0 +1,325 @@
+estimation_buffer: 0.5
+estimation_growth_factor: 0.01  # buffer increases by 1% per month
+storage_buffer: 0.33  # keep max storage at 75% of disk
+storage_display_unit: TB
+
+summary_dates:
+  - '2020-12'
+
+vm_os_storage_gb: 50
+vm_os_storage_group: 'VM_os'
+
+usage:
+  users:
+    model: 'date_range_value'
+    ranges:
+      - ['20180101', '20201201', 1132]
+
+  # LS User as a percent of total User
+  ls_users:
+    model: 'derived_factor'
+    dependant_field: 'users'
+    factor: 0.05
+  # Number of formaplayer user from X date to Y
+  formplayer_users:
+    model: 'date_range_value'
+    ranges:
+      - ['20180901', '20201201', 1000]
+
+  #Number of forms per user per month
+  forms_monthly:
+    model: 'derived_factor'
+    dependant_field: 'users'
+    factor: 0  # https://docs.google.com/spreadsheets/d/1cvgcihfE0oOrkAPW6RlmSLzGcsaD-3nDAOKVCYjyAeE/edit#gid=1011017089
```

Comment:
Can you remove this doc link its specific for ICDS

Also factor of 0 means that no one will be submitting new forms. Can you update this with the number of forms expected per user per month?